src/index.template.html

```diff
@@ -1,17 +1,18 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= productName %></title>
-    <meta name="description" content="<%= productDescription %>">
+    <title><%= title %></title>
+    <meta name="description" content="<%= description %>">
```

Comment:
This can stay as `productDescription` I think? Which gets it from `package.json` description field already.